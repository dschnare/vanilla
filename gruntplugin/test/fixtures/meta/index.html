<v:layout src="./page.html" />
<v:meta>
{
  title: function () {
    return this.url.split('/').pop().split('.').shift().toUpperCase();
  }
}
</v:meta>
<v:meta name="summary">
  This is the <strong>summary</strong>.
</v:meta>

<v:replace name="body">
  <p>This is the body!</p>
</v:replace>

<v:append name="body">
  <p>This is more content added to the body.</p>
</v:append>

<v:block name="any">
  this should be ignored.
</v:block>

<v:partial name="widget">
  <div>WIDGET</div>
</v:partial>