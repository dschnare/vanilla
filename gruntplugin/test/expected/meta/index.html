<html>
  <head>
    <title>INDEX</title><meta charset="utf-8" />
  </head>
  <body>
    This is the <strong>summary</strong>.
    <p>This is the body!</p><p>This is more content added to the body.</p>
    
    <div>WIDGET</div>    
    <div>
        <a href="/meta/index.html">INDEX</a>
    </div>
  </body>
</html>